common/src/main/resources/locales/ja-jp.yml

```diff
@@ -17,15 +17,16 @@ data_update_failed: '[🔔 Failed to update your data! Please contact an adminis
 data_manager_title: '[Viewing user data snapshot](#00fb9a) [%1%](#00fb9a show_text=&7Version UUID:\n&8%2%) [for](#00fb9a) [%3%](#00fb9a bold show_text=&7Player UUID:\n&8%4%)[:](#00fb9a)'
 data_manager_timestamp: '[⌚ %1%](#ffc43b-#f5c962 show_text=&7Version timestamp:\n&8When the data was saved)'
 data_manager_pinned: '[※ Snapshot pinned](#d8ff2b show_text=&7Pinned:\n&8This user data snapshot won''t be automatically rotated.)'
-data_manager_cause: '[⚑ %1%](#23a825-#36f539 show_text=&7Save cause:\n&8What caused the data to be saved)\n'
+data_manager_cause: '[⚑ %1%](#23a825-#36f539 show_text=&7Save cause:\n&8What caused the data to be saved)'
+data_manager_server: '[❖ %1%](#fb06c5-#E37dfd show_text=&7サーバー ID:\n&8スナップショットが作成されたサーバー)\n'
```

Comment:
For locales, best copy over the english locale -- leave TL up to contributors as MTL is unreliable